<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputBox</class>
 <widget class="QDialog" name="InputBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="table1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);
font: 10pt &quot;Impact&quot;;
background-color: rgb(209, 255, 171);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 0, 0);
	font: 28pt &quot;Impact&quot;;
}</string>
   </property>
   <property name="text">
    <string>Add your Income</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
font: 12pt &quot;Impact&quot;;
background-color: rgb(0, 85, 127);</string>
   </property>
   <item>
    <property name="text">
     <string>Salary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Savings</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Freelance </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Commission</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rental Income</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gifts</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
font: 12pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>Income Type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
font: 12pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="date1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Impact&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);
font: 12pt &quot;Impact&quot;;</string>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Impact&quot;;
background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Impact&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Impact&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Impact&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
